src/Components/Footer/Components/FooterTopMenu.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+import { FOOTERTOPDATA } from './data/FooterDatas';
```

Comment:
```suggestion
import { FOOTER_TOP_DATA } from './data/FooterData';
```
- 위와 같이 변수명 작성하는 것이 가독성 차원에서 좋을 것 같습니다
- data가 이미 복수형이기 때문에 s는 빼주세요!